Configurando el nodo 147.251.3.192
Configurando el nodo 147.251.3.195
Configurando el nodo 147.251.3.63
Configurando el nodo 147.251.3.54
Configurando el nodo 147.251.3.61
Configurando el nodo 147.251.3.194
Configurando el nodo 147.251.3.65
Configurando el nodo 147.251.3.230
Configurando el nodo 147.251.3.73
Configurando el nodo 147.251.3.231
Configurando el nodo 147.251.3.64
Configurando el nodo 147.251.3.232
Configurando el nodo 147.251.3.53
Configurando el nodo 147.251.3.106
Configurando el nodo 147.251.3.110
Configurando el nodo 147.251.3.123
Configurando el nodo 147.251.3.109
Configurando el nodo 147.251.3.111
Configurando el nodo 147.251.3.108
Configurando el nodo 147.251.3.90
Configurando el nodo 147.251.3.136
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@    WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!     @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
IT IS POSSIBLE THAT SOMEONE IS DOING SOMETHING NASTY!
Someone could be eavesdropping on you right now (man-in-the-middle attack)!
It is also possible that a host key has just been changed.
The fingerprint for the RSA key sent by the remote host is
66:5b:7b:da:81:09:5a:bb:e3:c6:ba:dc:56:78:34:65.
Please contact your system administrator.
Add correct host key in /mnt/home_common/jlopez_opensuse_13.1/.ssh/known_hosts to get rid of this message.
Offending RSA key in /mnt/home_common/jlopez_opensuse_13.1/.ssh/known_hosts:2343
You can use following command to remove all keys for this IP:
ssh-keygen -R 147.251.3.110 -f /mnt/home_common/jlopez_opensuse_13.1/.ssh/known_hosts
Password authentication is disabled to avoid man-in-the-middle attacks.
Keyboard-interactive authentication is disabled to avoid man-in-the-middle attacks.
... hadoop
Pseudo-terminal will not be allocated because stdin is not a terminal.
... hadoop
... hadoop
Pseudo-terminal will not be allocated because stdin is not a terminal.
Pseudo-terminal will not be allocated because stdin is not a terminal.
... hadoop
Pseudo-terminal will not be allocated because stdin is not a terminal.
... hadoop
... hadoop
Pseudo-terminal will not be allocated because stdin is not a terminal.
Pseudo-terminal will not be allocated because stdin is not a terminal.
... hadoop
... hadoop
... hadoop
Pseudo-terminal will not be allocated because stdin is not a terminal.
Pseudo-terminal will not be allocated because stdin is not a terminal.
Pseudo-terminal will not be allocated because stdin is not a terminal.
... hadoop
Pseudo-terminal will not be allocated because stdin is not a terminal.
... hadoop
Pseudo-terminal will not be allocated because stdin is not a terminal.
... hadoop
... hadoop
Pseudo-terminal will not be allocated because stdin is not a terminal.
... hadoop
Pseudo-terminal will not be allocated because stdin is not a terminal.
... hadoop
Pseudo-terminal will not be allocated because stdin is not a terminal.
Pseudo-terminal will not be allocated because stdin is not a terminal.
... hadoop
Pseudo-terminal will not be allocated because stdin is not a terminal.
... hadoop
... hadoop
... hadoop
Pseudo-terminal will not be allocated because stdin is not a terminal.
Pseudo-terminal will not be allocated because stdin is not a terminal.
... hadoop
... hadoop
Pseudo-terminal will not be allocated because stdin is not a terminal.
Pseudo-terminal will not be allocated because stdin is not a terminal.
Pseudo-terminal will not be allocated because stdin is not a terminal.
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@    WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!     @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
IT IS POSSIBLE THAT SOMEONE IS DOING SOMETHING NASTY!
Someone could be eavesdropping on you right now (man-in-the-middle attack)!
It is also possible that a host key has just been changed.
The fingerprint for the RSA key sent by the remote host is
66:5b:7b:da:81:09:5a:bb:e3:c6:ba:dc:56:78:34:65.
Please contact your system administrator.
Add correct host key in /mnt/home_common/jlopez_opensuse_13.1/.ssh/known_hosts to get rid of this message.
Offending RSA key in /mnt/home_common/jlopez_opensuse_13.1/.ssh/known_hosts:2343
You can use following command to remove all keys for this IP:
ssh-keygen -R 147.251.3.110 -f /mnt/home_common/jlopez_opensuse_13.1/.ssh/known_hosts
Password authentication is disabled to avoid man-in-the-middle attacks.
Keyboard-interactive authentication is disabled to avoid man-in-the-middle attacks.
 ... scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
 ... scratch
 ... scratch
 Ya existe /scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
 ... scratch
 ... scratch
 Ya existe /scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
 ... scratch
 Ya existe /scratch
14/06/09 15:07:39 INFO namenode.NameNode: STARTUP_MSG: 
/************************************************************
STARTUP_MSG: Starting NameNode
STARTUP_MSG:   host = hadoop-master/147.251.3.192
STARTUP_MSG:   args = [-format]
STARTUP_MSG:   version = 1.2.1
STARTUP_MSG:   build = https://svn.apache.org/repos/asf/hadoop/common/branches/branch-1.2 -r 1503152; compiled by 'mattf' on Mon Jul 22 15:23:09 PDT 2013
STARTUP_MSG:   java = 1.7.0_51
************************************************************/
14/06/09 15:07:40 INFO util.GSet: Computing capacity for map BlocksMap
14/06/09 15:07:40 INFO util.GSet: VM type       = 64-bit
14/06/09 15:07:40 INFO util.GSet: 2.0% max memory = 954728448
14/06/09 15:07:40 INFO util.GSet: capacity      = 2^21 = 2097152 entries
14/06/09 15:07:40 INFO util.GSet: recommended=2097152, actual=2097152
14/06/09 15:07:40 INFO namenode.FSNamesystem: fsOwner=hadoop
14/06/09 15:07:40 INFO namenode.FSNamesystem: supergroup=supergroup
14/06/09 15:07:40 INFO namenode.FSNamesystem: isPermissionEnabled=true
14/06/09 15:07:40 INFO namenode.FSNamesystem: dfs.block.invalidate.limit=100
14/06/09 15:07:40 INFO namenode.FSNamesystem: isAccessTokenEnabled=false accessKeyUpdateInterval=0 min(s), accessTokenLifetime=0 min(s)
14/06/09 15:07:40 INFO namenode.FSEditLog: dfs.namenode.edits.toleration.length = 0
14/06/09 15:07:40 INFO namenode.NameNode: Caching file names occuring more than 10 times 
14/06/09 15:07:41 INFO common.Storage: Image file /scratch/hadoop/dfs/name/current/fsimage of size 112 bytes saved in 0 seconds.
14/06/09 15:07:41 INFO namenode.FSEditLog: closing edit log: position=4, editlog=/scratch/hadoop/dfs/name/current/edits
14/06/09 15:07:41 INFO namenode.FSEditLog: close success: truncate to 4, editlog=/scratch/hadoop/dfs/name/current/edits
14/06/09 15:07:41 INFO common.Storage: Storage directory /scratch/hadoop/dfs/name has been successfully formatted.
14/06/09 15:07:41 INFO namenode.NameNode: SHUTDOWN_MSG: 
/************************************************************
SHUTDOWN_MSG: Shutting down NameNode at hadoop-master/147.251.3.192
************************************************************/
mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
starting namenode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-namenode-hadoop-master.out
/opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-namenode-hadoop-master.out: No such file or directory
head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-namenode-hadoop-master.out' for reading: No such file or directory
/opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-namenode-hadoop-master.out: No such file or directory
/opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-namenode-hadoop-master.out: No such file or directory
147.251.3.64: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.64: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.64: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-10.out
147.251.3.64: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-10.out: No such file or directory
147.251.3.64: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-10.out' for reading: No such file or directory
147.251.3.64: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-10.out: No such file or directory
147.251.3.64: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-10.out: No such file or directory
147.251.3.232: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.232: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.232: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-5.out
147.251.3.232: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-5.out: No such file or directory
147.251.3.232: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-5.out' for reading: No such file or directory
147.251.3.232: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-5.out: No such file or directory
147.251.3.232: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-5.out: No such file or directory
147.251.3.231: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.231: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.231: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-4.out
147.251.3.231: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-4.out: No such file or directory
147.251.3.231: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-4.out' for reading: No such file or directory
147.251.3.231: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-4.out: No such file or directory
147.251.3.231: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-4.out: No such file or directory
147.251.3.230: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.230: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.230: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-3.out
147.251.3.230: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-3.out: No such file or directory
147.251.3.230: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-3.out' for reading: No such file or directory
147.251.3.230: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-3.out: No such file or directory
147.251.3.230: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-3.out: No such file or directory
147.251.3.73: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.73: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.73: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-12.out
147.251.3.73: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-12.out: No such file or directory
147.251.3.73: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-12.out' for reading: No such file or directory
147.251.3.73: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-12.out: No such file or directory
147.251.3.73: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-12.out: No such file or directory
147.251.3.61: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.61: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.61: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-8.out
147.251.3.61: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-8.out: No such file or directory
147.251.3.61: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-8.out' for reading: No such file or directory
147.251.3.61: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-8.out: No such file or directory
147.251.3.61: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-8.out: No such file or directory
147.251.3.195: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.195: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.195: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-2.out
147.251.3.195: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-2.out: No such file or directory
147.251.3.195: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-2.out' for reading: No such file or directory
147.251.3.195: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-2.out: No such file or directory
147.251.3.195: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-2.out: No such file or directory
147.251.3.53: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.53: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.53: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-6.out
147.251.3.53: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-6.out: No such file or directory
147.251.3.53: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-6.out' for reading: No such file or directory
147.251.3.53: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-6.out: No such file or directory
147.251.3.53: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-6.out: No such file or directory
147.251.3.63: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.63: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.63: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-9.out
147.251.3.63: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-9.out: No such file or directory
147.251.3.63: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-9.out' for reading: No such file or directory
147.251.3.63: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-9.out: No such file or directory
147.251.3.63: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-9.out: No such file or directory
147.251.3.65: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.65: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.65: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-11.out
147.251.3.65: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-11.out: No such file or directory
147.251.3.65: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-11.out' for reading: No such file or directory
147.251.3.65: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-11.out: No such file or directory
147.251.3.65: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-11.out: No such file or directory
147.251.3.54: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.54: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.54: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-7.out
147.251.3.54: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-7.out: No such file or directory
147.251.3.54: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-7.out' for reading: No such file or directory
147.251.3.54: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-7.out: No such file or directory
147.251.3.54: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-7.out: No such file or directory
147.251.3.194: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.194: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.194: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-1.out
147.251.3.194: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-1.out: No such file or directory
147.251.3.194: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-1.out' for reading: No such file or directory
147.251.3.194: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-1.out: No such file or directory
147.251.3.194: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-1.out: No such file or directory
147.251.3.90: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.90: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.90: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-13.out
147.251.3.90: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-13.out: No such file or directory
147.251.3.90: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-13.out' for reading: No such file or directory
147.251.3.90: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-13.out: No such file or directory
147.251.3.90: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-13.out: No such file or directory
147.251.3.106: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.106: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.106: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-14.out
147.251.3.106: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-14.out: No such file or directory
147.251.3.106: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-14.out' for reading: No such file or directory
147.251.3.106: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-14.out: No such file or directory
147.251.3.106: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-14.out: No such file or directory
147.251.3.108: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.108: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.108: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-15.out
147.251.3.108: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-15.out: No such file or directory
147.251.3.108: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-15.out' for reading: No such file or directory
147.251.3.108: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-15.out: No such file or directory
147.251.3.108: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-15.out: No such file or directory
147.251.3.109: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.109: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.109: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-16.out
147.251.3.109: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-16.out: No such file or directory
147.251.3.109: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-16.out' for reading: No such file or directory
147.251.3.109: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-16.out: No such file or directory
147.251.3.109: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-16.out: No such file or directory
147.251.3.110: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.110: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.110: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-17.out
147.251.3.110: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-17.out: No such file or directory
147.251.3.110: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-17.out' for reading: No such file or directory
147.251.3.110: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-17.out: No such file or directory
147.251.3.110: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-17.out: No such file or directory
147.251.3.111: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.111: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.111: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-18.out
147.251.3.111: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-18.out: No such file or directory
147.251.3.111: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-18.out' for reading: No such file or directory
147.251.3.111: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-18.out: No such file or directory
147.251.3.111: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-18.out: No such file or directory
147.251.3.136: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.136: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.136: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-20.out
147.251.3.136: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-20.out: No such file or directory
147.251.3.136: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-20.out' for reading: No such file or directory
147.251.3.136: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-20.out: No such file or directory
147.251.3.136: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-20.out: No such file or directory
147.251.3.123: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.123: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.123: starting datanode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-19.out
147.251.3.123: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-19.out: No such file or directory
147.251.3.123: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-19.out' for reading: No such file or directory
147.251.3.123: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-19.out: No such file or directory
147.251.3.123: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-datanode-hadoop-19.out: No such file or directory
localhost: @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
localhost: @    WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!     @
localhost: @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
localhost: IT IS POSSIBLE THAT SOMEONE IS DOING SOMETHING NASTY!
localhost: Someone could be eavesdropping on you right now (man-in-the-middle attack)!
localhost: It is also possible that the RSA host key has just been changed.
localhost: The fingerprint for the RSA key sent by the remote host is
localhost: 10:6c:5e:d3:00:42:3a:00:cf:65:bf:8d:2b:56:02:01.
localhost: Please contact your system administrator.
localhost: Add correct host key in /home/hadoop/.ssh/known_hosts to get rid of this message.
localhost: Offending key in /home/hadoop/.ssh/known_hosts:1
localhost: Password authentication is disabled to avoid man-in-the-middle attacks.
localhost: Keyboard-interactive authentication is disabled to avoid man-in-the-middle attacks.
localhost: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
localhost: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
localhost: starting secondarynamenode, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-secondarynamenode-hadoop-master.out
localhost: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-secondarynamenode-hadoop-master.out: No such file or directory
localhost: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-secondarynamenode-hadoop-master.out' for reading: No such file or directory
localhost: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-secondarynamenode-hadoop-master.out: No such file or directory
localhost: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-secondarynamenode-hadoop-master.out: No such file or directory
mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
starting jobtracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-jobtracker-hadoop-master.out
/opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-jobtracker-hadoop-master.out: No such file or directory
head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-jobtracker-hadoop-master.out' for reading: No such file or directory
/opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-jobtracker-hadoop-master.out: No such file or directory
/opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-jobtracker-hadoop-master.out: No such file or directory
147.251.3.231: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.231: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.231: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-4.out
147.251.3.231: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-4.out: No such file or directory
147.251.3.231: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-4.out' for reading: No such file or directory
147.251.3.231: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-4.out: No such file or directory
147.251.3.231: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-4.out: No such file or directory
147.251.3.54: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.54: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.54: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-7.out
147.251.3.54: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-7.out: No such file or directory
147.251.3.54: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-7.out' for reading: No such file or directory
147.251.3.54: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-7.out: No such file or directory
147.251.3.54: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-7.out: No such file or directory
147.251.3.64: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.64: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.64: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-10.out
147.251.3.64: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-10.out: No such file or directory
147.251.3.64: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-10.out' for reading: No such file or directory
147.251.3.64: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-10.out: No such file or directory
147.251.3.64: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-10.out: No such file or directory
147.251.3.73: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.73: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.73: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-12.out
147.251.3.73: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-12.out: No such file or directory
147.251.3.73: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-12.out' for reading: No such file or directory
147.251.3.73: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-12.out: No such file or directory
147.251.3.73: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-12.out: No such file or directory
147.251.3.194: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.194: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.194: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-1.out
147.251.3.194: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-1.out: No such file or directory
147.251.3.194: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-1.out' for reading: No such file or directory
147.251.3.194: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-1.out: No such file or directory
147.251.3.194: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-1.out: No such file or directory
147.251.3.232: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.232: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.232: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-5.out
147.251.3.232: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-5.out: No such file or directory
147.251.3.232: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-5.out' for reading: No such file or directory
147.251.3.232: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-5.out: No such file or directory
147.251.3.232: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-5.out: No such file or directory
147.251.3.61: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.61: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.61: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-8.out
147.251.3.61: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-8.out: No such file or directory
147.251.3.61: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-8.out' for reading: No such file or directory
147.251.3.61: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-8.out: No such file or directory
147.251.3.61: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-8.out: No such file or directory
147.251.3.195: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.195: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.195: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-2.out
147.251.3.195: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-2.out: No such file or directory
147.251.3.195: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-2.out' for reading: No such file or directory
147.251.3.195: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-2.out: No such file or directory
147.251.3.195: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-2.out: No such file or directory
147.251.3.90: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.90: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.90: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-13.out
147.251.3.90: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-13.out: No such file or directory
147.251.3.90: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-13.out' for reading: No such file or directory
147.251.3.90: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-13.out: No such file or directory
147.251.3.90: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-13.out: No such file or directory
147.251.3.106: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.106: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.106: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-14.out
147.251.3.106: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-14.out: No such file or directory
147.251.3.106: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-14.out' for reading: No such file or directory
147.251.3.106: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-14.out: No such file or directory
147.251.3.106: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-14.out: No such file or directory
147.251.3.53: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.53: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.53: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-6.out
147.251.3.53: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-6.out: No such file or directory
147.251.3.53: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-6.out' for reading: No such file or directory
147.251.3.53: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-6.out: No such file or directory
147.251.3.53: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-6.out: No such file or directory
147.251.3.108: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.108: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.108: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-15.out
147.251.3.108: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-15.out: No such file or directory
147.251.3.108: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-15.out' for reading: No such file or directory
147.251.3.108: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-15.out: No such file or directory
147.251.3.108: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-15.out: No such file or directory
147.251.3.109: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.109: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.109: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-16.out
147.251.3.109: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-16.out: No such file or directory
147.251.3.109: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-16.out' for reading: No such file or directory
147.251.3.109: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-16.out: No such file or directory
147.251.3.109: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-16.out: No such file or directory
147.251.3.110: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.110: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.110: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-17.out
147.251.3.110: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-17.out: No such file or directory
147.251.3.110: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-17.out' for reading: No such file or directory
147.251.3.110: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-17.out: No such file or directory
147.251.3.110: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-17.out: No such file or directory
147.251.3.111: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.111: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.111: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-18.out
147.251.3.111: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-18.out: No such file or directory
147.251.3.111: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-18.out' for reading: No such file or directory
147.251.3.111: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-18.out: No such file or directory
147.251.3.111: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-18.out: No such file or directory
147.251.3.123: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.123: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.123: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-19.out
147.251.3.123: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-19.out: No such file or directory
147.251.3.123: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-19.out' for reading: No such file or directory
147.251.3.123: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-19.out: No such file or directory
147.251.3.123: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-19.out: No such file or directory
147.251.3.136: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.136: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.136: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-20.out
147.251.3.136: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-20.out: No such file or directory
147.251.3.136: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-20.out' for reading: No such file or directory
147.251.3.136: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-20.out: No such file or directory
147.251.3.136: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-20.out: No such file or directory
147.251.3.63: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.63: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.63: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-9.out
147.251.3.63: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-9.out: No such file or directory
147.251.3.63: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-9.out' for reading: No such file or directory
147.251.3.63: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-9.out: No such file or directory
147.251.3.63: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-9.out: No such file or directory
147.251.3.65: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.65: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.65: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-11.out
147.251.3.65: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-11.out: No such file or directory
147.251.3.65: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-11.out' for reading: No such file or directory
147.251.3.65: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-11.out: No such file or directory
147.251.3.65: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-11.out: No such file or directory
147.251.3.230: mkdir: cannot create directory `/opt/cesga/hadoop-1.2.1/libexec/../logs': File exists
147.251.3.230: chown: cannot dereference `/opt/cesga/hadoop-1.2.1/libexec/../logs': No such file or directory
147.251.3.230: starting tasktracker, logging to /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-3.out
147.251.3.230: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 137: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-3.out: No such file or directory
147.251.3.230: head: cannot open `/opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-3.out' for reading: No such file or directory
147.251.3.230: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 147: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-3.out: No such file or directory
147.251.3.230: /opt/cesga/hadoop/bin/hadoop-daemon.sh: line 148: /opt/cesga/hadoop-1.2.1/libexec/../logs/hadoop-hadoop-tasktracker-hadoop-3.out: No such file or directory
14/06/09 15:08:09 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:10 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:11 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:12 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:13 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:14 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:15 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:16 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:17 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:18 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
mkdir: Call to hadoop-master/147.251.3.192:54310 failed on connection exception: java.net.ConnectException: Connection refused

-------------- HADOOP STATUS ------------------------------
==> Hadoop cluster status
14/06/09 15:08:32 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:33 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:34 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:35 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:36 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:37 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:38 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:39 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:40 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:41 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
report: Call to hadoop-master/147.251.3.192:54310 failed on connection exception: java.net.ConnectException: Conexión rehusada
Tasktrackers available 14/06/09 15:08:43 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:44 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:45 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:46 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:47 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:48 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:49 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:50 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:51 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:52 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
Exception in thread "main" java.net.ConnectException: Call to hadoop-master/147.251.3.192:54311 failed on connection exception: java.net.ConnectException: Conexión rehusada
	at org.apache.hadoop.ipc.Client.wrapException(Client.java:1142)
	at org.apache.hadoop.ipc.Client.call(Client.java:1118)
	at org.apache.hadoop.ipc.RPC$Invoker.invoke(RPC.java:229)
	at org.apache.hadoop.mapred.$Proxy1.getProtocolVersion(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:85)
	at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:62)
	at org.apache.hadoop.mapred.$Proxy1.getProtocolVersion(Unknown Source)
	at org.apache.hadoop.ipc.RPC.checkVersion(RPC.java:422)
	at org.apache.hadoop.mapred.JobClient.createProxy(JobClient.java:559)
	at org.apache.hadoop.mapred.JobClient.init(JobClient.java:498)
	at org.apache.hadoop.mapred.JobClient.run(JobClient.java:1786)
	at org.apache.hadoop.util.ToolRunner.run(ToolRunner.java:65)
	at org.apache.hadoop.util.ToolRunner.run(ToolRunner.java:79)
	at org.apache.hadoop.mapred.JobClient.main(JobClient.java:2123)
Caused by: java.net.ConnectException: Conexión rehusada
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:739)
	at org.apache.hadoop.net.SocketIOWithTimeout.connect(SocketIOWithTimeout.java:206)
	at org.apache.hadoop.net.NetUtils.connect(NetUtils.java:511)
	at org.apache.hadoop.net.NetUtils.connect(NetUtils.java:481)
	at org.apache.hadoop.ipc.Client$Connection.setupConnection(Client.java:457)
	at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:583)
	at org.apache.hadoop.ipc.Client$Connection.access$2200(Client.java:205)
	at org.apache.hadoop.ipc.Client.getConnection(Client.java:1249)
	at org.apache.hadoop.ipc.Client.call(Client.java:1093)
	... 16 more
0
===> HDFS
14/06/09 15:08:55 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:56 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:57 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:58 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:08:59 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:09:00 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:09:01 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:09:02 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:09:03 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:09:04 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54310. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
report: Call to hadoop-master/147.251.3.192:54310 failed on connection exception: java.net.ConnectException: Conexión rehusada
===> Task trackers
14/06/09 15:09:06 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 0 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:09:07 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 1 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:09:08 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 2 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:09:09 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 3 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:09:10 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 4 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:09:11 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 5 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:09:12 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 6 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:09:13 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 7 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:09:14 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 8 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
14/06/09 15:09:15 INFO ipc.Client: Retrying connect to server: hadoop-master/147.251.3.192:54311. Already tried 9 time(s); retry policy is RetryUpToMaximumCountWithFixedSleep(maxRetries=10, sleepTime=1 SECONDS)
Exception in thread "main" java.net.ConnectException: Call to hadoop-master/147.251.3.192:54311 failed on connection exception: java.net.ConnectException: Conexión rehusada
	at org.apache.hadoop.ipc.Client.wrapException(Client.java:1142)
	at org.apache.hadoop.ipc.Client.call(Client.java:1118)
	at org.apache.hadoop.ipc.RPC$Invoker.invoke(RPC.java:229)
	at org.apache.hadoop.mapred.$Proxy1.getProtocolVersion(Unknown Source)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.apache.hadoop.io.retry.RetryInvocationHandler.invokeMethod(RetryInvocationHandler.java:85)
	at org.apache.hadoop.io.retry.RetryInvocationHandler.invoke(RetryInvocationHandler.java:62)
	at org.apache.hadoop.mapred.$Proxy1.getProtocolVersion(Unknown Source)
	at org.apache.hadoop.ipc.RPC.checkVersion(RPC.java:422)
	at org.apache.hadoop.mapred.JobClient.createProxy(JobClient.java:559)
	at org.apache.hadoop.mapred.JobClient.init(JobClient.java:498)
	at org.apache.hadoop.mapred.JobClient.run(JobClient.java:1786)
	at org.apache.hadoop.util.ToolRunner.run(ToolRunner.java:65)
	at org.apache.hadoop.util.ToolRunner.run(ToolRunner.java:79)
	at org.apache.hadoop.mapred.JobClient.main(JobClient.java:2123)
Caused by: java.net.ConnectException: Conexión rehusada
	at sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:739)
	at org.apache.hadoop.net.SocketIOWithTimeout.connect(SocketIOWithTimeout.java:206)
	at org.apache.hadoop.net.NetUtils.connect(NetUtils.java:511)
	at org.apache.hadoop.net.NetUtils.connect(NetUtils.java:481)
	at org.apache.hadoop.ipc.Client$Connection.setupConnection(Client.java:457)
	at org.apache.hadoop.ipc.Client$Connection.setupIOstreams(Client.java:583)
	at org.apache.hadoop.ipc.Client$Connection.access$2200(Client.java:205)
	at org.apache.hadoop.ipc.Client.getConnection(Client.java:1249)
	at org.apache.hadoop.ipc.Client.call(Client.java:1093)
	... 16 more
-------------- HADOOP STATUS ------------------------------
lun jun  9 15:14:38 CEST 2014
